packages/app/src/app/components/CodeEditor/CodeMirror/index.js

```diff
@@ -111,6 +112,11 @@ class CodemirrorEditor extends React.Component<Props, State> implements Editor {
     });
   }
 
+  resizeEditor() {
+    // eslint-disable-next-line no-unused-expressions
+    this.codemirror?.refresh();
```

Comment:
wouldn't this always exist on codemirror 🤔

Reply:
There may be the case where `codemirror` is null, so this is to be on the safe side there.